content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Memory Manager
+
+reviewers:
+- klueska
+- derekwaynecarr
+- TBD
+
+content_type: task
+min-kubernetes-server-version: v1.21
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.21" >}}
+
+The Kubernetes *Memory Manager* enables the feature of guaranteed memory (and hugepages) allocation for pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}.
+
+The Memory Manager employs hint generation protocol to yield the most suitable NUMA affinity for a pod. The Memory Manager feeds the central manager (*Topology Manager*) with these affinity hints. Based on both the hints and Topology Manager policy, the pod is rejected or admitted to the node.
+
+Moreover, the Memory Manager ensures that the memory which a pod requests is allocated from a minimum number of NUMA nodes.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+To align memory resources with other requested resources in a Pod Spec:
+- the CPU Manager should be enabled and proper CPU Manager policy should be configured on a Node. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/);
+- the Topology Manager should be enabled and proper Topology Manager policy should be configured on a Node. See [control Topology Management Policies](/docs/tasks/administer-cluster/topology-manager/).
+
+Support for the Memory Manager requires `MemoryManager` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. 
+
+That is, the `kubelet` must be started with the following flag:
+
+`--feature-gates=MemoryManager=true`
+
+## How Memory Manager Operates?
+
+The Memory Manager currently offers the guaranteed memory (and hugepages) allocation for Pods in Guaranteed QoS class. To immediately put the Memory Manager into operation follow the guidelines in section [Memory Manager configuration](#memory-manager-configuration)), and subsequently, prepare and deploy a `Guaranteed` pod as illustrated in section [Placing a Pod in the Guaranteed QoS class](placing-a-pod-in-the-guaranteed-qos-class).
+
+The Memory Manager is a Hint Provider, and it provides topology hints for the Topology Manager which then aligns the requested resources according to these topology hints. It also enforces `cgroups` (i.e. `cpuset.mems`) for pods. The complete flow diagram concerning pod admission and deployment process is illustrated in [Memory Manager KEP: Design Overview][4] and below:
+
+![Memory Manager in the pod admission and deployment process](memory-manager/mm_diag.svg)

```

Comment:
This image isn't rendering. Incorrect hyperlink?

Reply:
Please check here: https://github.com/kubernetes/website/blob/0f02d8bb4bdcdcbde0686a4cd8a9e38698567dd6/content/en/docs/tasks/administer-cluster/memory-manager.md